contracts/testing/CryptoPool/CurveCryptoLP.vy

```diff
@@ -0,0 +1,221 @@
+# @version 0.2.12
```

Comment:
I added a tag to the `curve-crypto-contract` repo: https://github.com/curvefi/curve-crypto-contract/releases/tag/v1.0.0

Prefer you to use that then to add all this stuff under `testing/` as this puts considerable overhead in compiling the repo.